birdhouse/components/stac/default.env

```diff
@@ -3,6 +3,20 @@ export STAC_POSTGRES_PASSWORD=${POSTGRES_PAVICS_PASSWORD}
 export STAC_PGUSER=${POSTGRES_PAVICS_USERNAME}
 export STAC_PGPASSWORD=${POSTGRES_PAVICS_PASSWORD}
 
+# 'main' branch points at https://github.com/stac-utils/stac-fastapi/commit/d53e792
+# (see: https://github.com/crim-ca/stac-app/blob/40cad1aa7a094d58fca2d3184182761e248f781d/Dockerfile#L15-L20)
+# which corresponds to 2.4.3 release
+export STAC_VERSION=2.4.3-crim-main
+export STAC_IMAGE='ghcr.io/crim-ca/stac-app:main'
+export STAC_IMAGE_URI='${STAC_IMAGE}'
+
+# 'docker_image_push' branch points at https://github.com/crim-ca/stac-browser/tree/docker_image_push
+# which is some commits ahead of 'v3.0.0-beta.5' (and multiple behind latest official releases)
+# version name is slightly tweaked to fulfill schema while leaving an obvious trace
+export STAC_BROWSER_VERSION=3.0.0-beta.5-crim-docker-image-push
+export STAC_BROWSER_IMAGE='ghcr.io/crim-ca/stac-browser:docker_image_push'
```

Comment:
Same here, `3.0.0-beta.5-crim` instead of `docker_image_push`.  It is allowed to tag while on a branch.